docs/pools.adoc

```diff
@@ -56,3 +101,53 @@ Sample Response
 Status: 202 Accepted
 { job_id: "3784922e33e8bec939be5e626e21a174" }
 ----------
+
+== RBD's
+
+=== Create RBD
+
+Sample Request
+----------
+curl -XPOST -d
+'{"Rbd.pool_id":"10","Rbd.name":"rbd_001","Rbd.size":1024}'
```

Comment:
@anivargi How the below details will be sent in POST call for creating RBD?

- Number of RBDs to create
- Cluster Name to which it will be associated
- Target Size